.github/dco.yml

```diff
@@ -0,0 +1,2 @@
+require:
+  members: false
```

Comment:
Is the DCO reallu necessary? I do not remember getting these annouying messages when submitting patches to Ansible. Or this is a new legal requirement?